cumulus/parachains/common/src/xcm_config.rs

```diff
@@ -1,3 +1,18 @@
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Wha is this one apache?

Reply:
This one I made Apache simply because the files right next to it were Apache: https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/common/src/lib.rs

But why the licenses are mixed throughout the project, I don't know.

Reply:
Do you know @bkchr?  

I guess it is better than no license, but eventually we should try to have the license checker verify that the license that is declared in the crate manifest is the same as in all files.

Reply:
It's actually described in the [substrate readme](https://github.com/paritytech/polkadot-sdk/tree/master/substrate#license) - as pointed out by @gilescope